drivers/wdm/audio/drivers/ac97/prophnd.cpp

```diff
@@ -171,6 +171,13 @@ NTSTATUS CAC97MiniportTopology::GetDBValues
         // These nodes could have 5bit or 6bit controls, so we first
         // have to check this.
         case NODE_MASTEROUT_VOLUME:
```

Comment:
It will be simpler by `ifndef`'ing this one, and `ifdef`'ing it below near the `NODE_WAVEOUT_VOLUME`.

Reply:
Indeed! Thanks!

Reply:
Although as I got a reply from the chat, it is not necessarily to use ifdefs since this will not be synced in the future...

Reply:
Done. :slightly_smiling_face:

Reply:
If ifdefing is not necessary, you can go with a one-liner diff, moving a case.